src/main/scala/se/nimsa/sbx/importing/ImportDAO.scala

```diff
@@ -46,4 +46,23 @@ class ImportDAO(val driver: JdbcProfile) {
     importSessionQuery.delete //Cascade deletes ImportSessionImages
   }
 
+  def getImportSessions(implicit session: Session): List[ImportSession] = {
```

Comment:
Should be named listImportSessions to conform to naming in other DAOs in Slicebox.